docs/docker-dev/docker-dev-setup-manual.md

```diff
@@ -0,0 +1,18 @@
+# Docker Development Environment Setup
+
+1. Install [Docker](https://docs.docker.com/get-docker/) if not already.
```

Comment:
```suggestion
1. Install [Docker](https://docs.docker.com/get-docker/) if not already installed.
```

not required, but it can be useful in such cases to provide a quick command to check.